corn-maze.lic

```diff
@@ -0,0 +1,632 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#corn-maze
+  This script runs non-combat tasks for sleeping dragon maze. 
```

Comment:
💡 Maybe 'This script is intended to replace loot-corn-maze.lic`?

Reply:
loot-corn-maze is just the looting part ripped out of this script, its still useful if you!re manually running around the maze

Reply:
Okay, I never saw loot-corn-maze somehow.  It looks useful, but also like it's totally obsoleted by this... although it'd be pretty trivial to add a 'manual' arg to this that does the same thing.

I guess the question is, should this replace loot-corn-maze entirely?  I like corn-maze.lic more than loot-corn-maze, but that's just like my opinion, man.